.github/workflows/test-changes.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           python -m pip install -r requirements-formats.txt
           echo "::endgroup::"
           echo "::group::Perform doc test execution with coverage"
-          nosetests -v --with-coverage --cover-package=petl --with-doctest --doctest-options=+NORMALIZE_WHITESPACE petl -I"csv_py2\.py" -I"db\.py"
+          pytest --cov=petl petl
```

Comment:
I was wondering what would be the best approach here:

- Calling pytest directly from here as I did with nose, or...
- Calling pytest through tox as one would in command line

Not a requirement, off course. Just getting any possible opinion on the subject.

Reply:
For now, I would leave it as is, calling pytest directly. Ideally, however I would prefer to align the call as one would do it locally. I recall that in one of my previous PRs there were some slight discrepancies between calling `tox` locally and how one of the functions behaved on CI.
I would even consider a Docker approach, or  docker-compose with db and remote services (instead of calling docker run)